PTT/transformers.py

```diff
@@ -1,3 +1,4 @@
+import re
```

Comment:
Don't use re, use regex library instead. It has the same as re lib but also offers more functionality. Regex is an enhanced version of the re lib. Re is like the watered down version that comes with python but doesn't support some of the stuff we need.